### ISSUE TITLE
Switch from psycopg2 to psycopg2-binary

### ISSUE THREAD

**Issue by mjalkio** [SimpleFinance/shiftmanager#41]
The latest version of `psycopg2` (2.7.4) throws a warning when it's installed from wheel. I think that's preferred, and their suggestion is to switch to `psycopg2-binary` to continue to do that.

The issue I'm having right now is that `setup.py` specifies >=2.5.4, and that causes it to install the latest version. I'll try to workaround for now, but I think either making the switch or locking the version to <2.7.4 would be okay.

Warning:
```
The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
```

Release notes: http://initd.org/psycopg/docs/news.html#what-s-new-in-psycopg-2-7-4

**Comment by jklukas** [comment]
My reading of http://initd.org/psycopg/docs/install.html#disabling-wheel-packages-for-psycopg-2-7 is that disabling the wheel binary download is something that can be specified in the `requirements.txt` or `pip install` invocation.

So, a project consuming shiftmanager would specify `--no-binary` for psycopg2 in requirements.txt and thus avoid the wheel and the warning message.

It sounds like this will become a non-issue once 2.8 is released; if 2.8 is pulled down, it'll be built from source, which is recommended as the safest option.

My take is that it's reasonable to ask consumers of shiftmanager to specify psycopg2 version to get the behavior they want for now, and we'll default to the binary install in the future. I don't claim to fully and deeply understand the implications of the wheel/binary install options, though.

**Comment by mjalkio** [comment]
I agree that building from source using `--no-binary` would avoid the issue, but I think that most people prefer the simpler wheel setup.

I think getting more specific in your `setup.py` requirement would be helpful (so that people that install through that get the same version that's specified in `requirements.txt`). But for my project we've already specified the version to workaround this warning.

I'm also a bit lost in this world of requirements, thanks for looking into it :)
